docs/user-guide/images.md

```diff
@@ -100,6 +101,25 @@ Troubleshooting:
   - `plugins.go:56] Registering credential provider: aws-ecr-key`
   - `provider.go:91] Refreshing cache for provider: *aws_credentials.ecrProvider`
 
+### Using Azure Container Registry (ACR)
+When using [Azure Container Registry](https://azure.microsoft.com/en-us/services/container-registry/)
+you can authenticate using either an admin user or a service principal.
+In either case, authentication is done via standard Docker authentication.  These instructions assume the
+[azure-cli](https://github.com/azure/azure-cli) command line tool.
+
+You first need to create a registry and generate credentials, complete documentation for this can be found in
+the [Azure container registry documentation](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-azure-cli)
+
+Once you have created your container registry, you will use the following credentials to login:
+   * `DOCKER_USER` : service principal, or admin username
+   * `DOCKER_PASSWORD`: service principal password, or admin user password
+   * `DOCKER_REGISTRY_SERVER`: `${some-registry-name}.azurecr.io`
+   * `DOCKER_EMAIL`: `${some-email-address}`
+
+Once you have those variables filled in you can [configure a Kubernetes Secret and use it to deploy a Pod]
```

Comment:
This link does not render correctly. Check to see if you don't have a whitespace issue.